multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/data/NDArray.kt

```diff
@@ -25,7 +25,8 @@ public class NDArray<T, D : Dimension> constructor(
     public override val shape: IntArray,
     public override val strides: IntArray = computeStrides(shape),
     public override val dtype: DataType,
```

Comment:
Why dtype remained in NDArray constructor?

Reply:
Removed `dtype` parameter in d2358d7